themes/scss/map/overlays.scss

```diff
@@ -272,6 +272,14 @@ $maxLegendContainerHeight: 300px;
   margin-bottom: 12px;
 }
 
+
+.CDB-Legend-item div.cartodb-legend {
```

Comment:
yes we only need override those properties

![image](https://cloud.githubusercontent.com/assets/1214201/19764666/7d18f2c2-9c44-11e6-929b-2d627c7b9652.png)